File: busqueda_lineal.py
#funcion de busqueda lineal
def busqueda_lineal(arr, buscar):
    for i in range(len(arr)):
        if arr[i] == buscar:
            return i
    return -1

# Ejemplo de uso
arr = [3, 6, 8, 10, 1, 2, 1]
print("Arreglo:", arr)
buscar = 10
indice = busqueda_lineal(arr, buscar)
if indice != -1:
    print(f"Elemento {buscar} encontrado en el índice: {indice + 1}.")
else:
    print(f"Elemento {buscar} no encontrado en el arreglo.")

File: busqueda_lineal.js
//busqueda lineal
function busquedaLineal(arr, objetivo) {
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === objetivo) {
            return i; // Retorna el índice si se encuentra el objetivo
        }
    }
    return -1; // Retorna -1 si no se encuentra el objetivo
}

// Ejemplo de uso:
const arreglo = [4, 2, 7, 1, 3];
const objetivo = 7;
const resultado = busquedaLineal(arreglo, objetivo);
if (resultado !== -1) {
    console.log(`Elemento encontrado en el índice: ${resultado + 1}`);
} else {
    console.log('Elemento no encontrado en el arreglo.');
}

return 0;
